src/common.c

```diff
@@ -74,7 +74,11 @@ char *get_config_path(char* default_path, char* pbuf, size_t bufsize) {
 
 	// priority 3; $HOME/.proxychains/proxychains.conf
 	path = getenv("HOME");
+#ifdef IS_HAIKU
+	snprintf(pbuf, bufsize, "%s/config/settings/%s", path, PROXYCHAINS_CONF_FILE);
```

Comment:
i'd rather just add this path unconditionally as "priority 4" rather than having distrinct layout per OS